tests/scripts/perf-prep.sh

```diff
@@ -68,7 +68,7 @@ python3.5 ./tests/scripts/Microsoft.BenchView.JSONFormat/tools/machinedata.py
 
 if [ $throughput -eq 1 ]; then
     # Download throughput benchmarks
-    if [ -d "Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT" ]; then
+    if [ ! -d "Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT" ]; then
         mkdir Microsoft.Benchview.ThroughputBenchmarks.x64.Windows_NT
```

Comment:
Does it matter that we don't leave the directory afterwards, or is does that not matter to the script which invoked this one?

Reply:
Hm, it has not mattered in the past, but I guess that would make it cleaner.

Reply:
Oh, ok. So this is code that's been running for a while. I checked out the call sites and ya, it's clearly not affecting anything